spec/features/ctc/admin_tools_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-RSpec.feature "CTC Admin Tools", active_job: true do
```

Comment:
The `#rename` comment should probably be deleted.

Reply:
oops good catch!